data/campaigns/Eastern_Invasion/scenarios/07a_Capturing_the_Ogres.cfg

```diff
@@ -197,7 +238,7 @@
 
         [message]
             speaker=Gweddry
-            message= _ "We have let one escape. Let’s hope they don’t all!"
+            message= _ "We have let one escape. Let’s hope they all don’t!"
```

Comment:
While the original sounds off, this one doesn't sound right either, since one has already run away. How about 'Let's hope not all do!'

Reply:
Let's hope they don't all evade us!

Or, with a `#po: a metaphorical rather than physical net`
One has slipped the net. Let's hope they don't all escape!